helsingborg-stad/app-mitt-helsingborg
# feat(viva-period): add VivaPeriodService and VivaPeriodCard

## Explain the changes you’ve made

Added VivaPeriodService to fetch VIVA Period status message, and VivaCaseCard that displays the status message, if any.

## Explain why these changes are made

So users can see when their VIVA period opens if not already opened.

## Explain your solution

Added Service to work with API. Added Context to work with Service in React. Added new component that wraps CaseCard and use it in CaseOverview.

## How to test

Concrete example:

1. Checkout this branch
2. Login
3. Verify new grayed-out card shows if period is not opened yet, with text

## Tested environments

- [] Storybook on a iOS device/simulator.
- [] Building the Application on a Android device/simulator.

## Screenshots

<img width="472" alt="image" src="https://user-images.githubusercontent.com/2890987/216656980-0763fae7-c819-4435-b420-c6c96951abc3.png">
